src/lib/utils/thorchain/memo/addFinalAssetContractToMemo.ts

```diff
@@ -0,0 +1,59 @@
+import { MEMO_PART_DELIMITER } from './constants'
+
+type AddLimitToMemoArgs = {
```

Comment:
```suggestion
type AddFinalAssetContractToMemoArgs = {
```

Reply:
in [2a6fc6a](https://github.com/shapeshift/web/pull/7146/commits/2a6fc6a57521f17fb2fa9ee67d3d46f222c6e53c)